deltachat-ios/Controller/GalleryViewController.swift

```diff
@@ -183,12 +184,14 @@ extension GalleryViewController: UICollectionViewDataSource, UICollectionViewDel
         var item: GalleryItem
         if let galleryItem = items[indexPath.row] {
             item = galleryItem
+            galleryCell.update(item: item)
         } else {
-            let galleryItem = GalleryItem(msgId: msgId)
+            let message = dcContext.getMessage(id: msgId)
+            let galleryItem = GalleryItem(msg: message)
             items[indexPath.row] = galleryItem
             item = galleryItem
+            galleryCell.update(item: item)
         }
-        galleryCell.update(item: item)
```

Comment:
why is `galleryCell.update(item: item)` no longer outfactored?

Reply:
that's not how it should be. I'll correct it